### DISCUSSION TITLE
Usage with SuperSlicer

### DISCUSSION THREAD

**Discussion by Leafy1** [nicolai-wachenschwan/arc-overhang-prusaslicer-integration#17] [Q&A] [answered]
So added this to super slicer, the console window pops up but doesn't show much of anything happening before it closes and I get a type 1 error. I ran it manuall in power shell and it get TypeError: unsupported operand type(s) for *: 'float' and 'NoneType'
 Is my g-code file not right because I'm using superslicer instead of prusa slicer or is there something wrong with my python install?

**Comment by nicolai-wachenschwan** [comment, marked as answer]
This arrised, because it is made for PRUSA Slicer. It looks for the line ";prusaslicer_config=begin" in the gcode, else raises error. Other users had the same issue and reported it, therefore with the current version the keyword is adapted and the error should be fixed.

**Reply by Leafy1** [reply]
Thankyou very much.
